### PR TITLE
Add adjective and examples to one file per namespace

### LINKED ISSUE #145: Unclear what this means: Use one file per namespace.
Hi,

The guide says: `Use one file per namespace.`, but I'm unclear if this means don't have more than one namespace per file, or if it means don't break a namespace over multiple files?

### PR DESCRIPTION
**Pull request by SherSpock**
Fixes #145

This was basically written by the existing discussion on that issue

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -339,7 +339,17 @@ Avoid trailing whitespace.
 
 === One File Per Namespace [[one-file-per-namespace]]
 
-Use one file per namespace.
+Use exactly one file per namespace.
```
Comment (bbatsov) [inline README.adoc]: I guess there are two aspects to this - having a namespace over multiple files and having several namespaces in the same file (e.g. using `in-ns`).

Comment (SherSpock) [reply README.adoc]: Gotcha. I think I understand the direction things should change in

**Comment by SherSpock** [comment]
I pushed some changes that should represent an improvement.

I am operating from the fallible understanding that `in-ns` works the same as `ns` without pulling in `clojure.core`

Is this more suitable?

**Comment by bbatsov** [comment]
Yeah, I think that's a solid improvement.

**Pull request merged**